docs/dev/swagger.yaml

```diff
@@ -340,6 +396,35 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/Galley'
+  IssueSummary:
+    type: object
+    properties:
+      id:
+        type: integer
+      _href:
+        type: string
+      title:
+        $ref: '#/definitions/LocaleObject'
+      description:
+        $ref: '#/definitions/LocaleObject'
+      identification:
+        type: string
+      volume:
+        type: integer
+      number:
```

Comment:
Hmm, this may be one of those cases where we should coerce type. @asmecher would an Issue Number ever not be an `int`?